<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="gameWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="panWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="pieceTree">
            <column>
             <property name="text">
              <string>point</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>moveNum</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="processWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>研究</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="storeBtn">
         <property name="text">
          <string>存储</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="LoadBtn">
         <property name="text">
          <string>读取</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="WinBtn">
         <property name="text">
          <string>判定胜负</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="text">
          <string>读取定式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_27">
         <property name="text">
          <string>近似定式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_4">
         <property name="text">
          <string>试下</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="clearBtn">
         <property name="text">
          <string>清空</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>reset</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="judgeBtn">
         <property name="text">
          <string>形式判断</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_10">
         <property name="text">
          <string>智能裁判</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_11">
         <property name="text">
          <string>智能分析</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="addDSBtn">
         <property name="text">
          <string>入库</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>比赛</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_6">
         <property name="text">
          <string>悔棋</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_8">
         <property name="text">
          <string>和棋</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_19">
         <property name="text">
          <string>停一手</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_9">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_7">
         <property name="text">
          <string>认输</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="testWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>黑白</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_20">
         <property name="text">
          <string>手数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_21">
         <property name="text">
          <string>移动</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_22">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_23">
         <property name="text">
          <string>摆黑</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_24">
         <property name="text">
          <string>摆白</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_25">
         <property name="text">
          <string>交替</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="watchWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>进度条</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="Begin">
         <property name="text">
          <string>|&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="leftFive">
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="leftOne">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="autoPlay">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>▶</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="rightOne">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="rightFive">
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="End">
         <property name="text">
          <string>|&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pieceEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>跳转</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionduqu"/>
    <addaction name="actioncunchu"/>
    <addaction name="actionshezhi"/>
    <addaction name="actionyonghu"/>
    <addaction name="separator"/>
    <addaction name="actiontuichu"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionundo"/>
    <addaction name="actionredo"/>
    <addaction name="actionjump"/>
    <addaction name="actiondelete"/>
    <addaction name="actiontry"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionjudgeWin"/>
    <addaction name="actionjudgeSituation"/>
    <addaction name="actionAIAnalysis"/>
    <addaction name="actionnextStep"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="actiontheme"/>
    <addaction name="actionsearchStep"/>
    <addaction name="actionshezhi_2"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="actionguanyu"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionguanyu">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>帮助内容</string>
   </property>
  </action>
  <action name="actionduqu">
   <property name="text">
    <string>读取</string>
   </property>
  </action>
  <action name="actioncunchu">
   <property name="text">
    <string>存储</string>
   </property>
  </action>
  <action name="actiontuichu">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionshezhi">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionyonghu">
   <property name="text">
    <string>用户</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionredo">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="actionjump">
   <property name="text">
    <string>跳转</string>
   </property>
  </action>
  <action name="actiondelete">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actiontry">
   <property name="text">
    <string>试下</string>
   </property>
  </action>
  <action name="actionsearchStep">
   <property name="text">
    <string>设置搜索步数</string>
   </property>
  </action>
  <action name="actiontheme">
   <property name="text">
    <string>主题</string>
   </property>
  </action>
  <action name="actionjudgeWin">
   <property name="text">
    <string>判定胜负</string>
   </property>
  </action>
  <action name="actionjudgeSituation">
   <property name="text">
    <string>形势判断</string>
   </property>
  </action>
  <action name="actionAIAnalysis">
   <property name="text">
    <string>智能分析</string>
   </property>
  </action>
  <action name="actionnextStep">
   <property name="text">
    <string>定式下一步</string>
   </property>
  </action>
  <action name="actionshezhi_2">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
